x/protocolpool/keeper/keeper.go

```diff
@@ -90,6 +101,138 @@ func (k Keeper) GetCommunityPool(ctx context.Context) (sdk.Coins, error) {
 	return k.bankKeeper.GetAllBalances(ctx, moduleAccount.GetAddress()), nil
 }
 
+func (k Keeper) withdrawContinuousFund(ctx context.Context, recipient sdk.AccAddress) (sdk.Coin, error) {
+	cf, err := k.ContinuousFund.Get(ctx, recipient)
+	if err != nil {
+		if errors.Is(err, collections.ErrNotFound) {
+			return sdk.Coin{}, fmt.Errorf("no continuous fund found for recipient: %s", recipient.String())
+		}
+		return sdk.Coin{}, err
+	}
+	denom, err := k.iterateAndUpdateFundsDistribution(ctx)
+	if err != nil {
+		return sdk.Coin{}, fmt.Errorf("error while iterating all the continuous funds: %w", err)
+	}
+	// get allocated continuous fund
+	fundsAllocated, err := k.getAndResetDistributedFunds(ctx, recipient)
+	if err != nil {
+		return sdk.Coin{}, fmt.Errorf("error while getting distributed funds: %w", err)
+	}
+
+	recipientBal := k.bankKeeper.GetAllBalances(ctx, recipient)
+	recipientAmount := recipientBal.AmountOf(denom)
+	totalRecipientBal := recipientAmount.Add(fundsAllocated)
+	// check if the recipient account balance exceeds maxDistributedCapital after distribution
+	var withdrawnAmount sdk.Coin
+	if totalRecipientBal.LT(cf.MaxDistributedCapital) {
```

Comment:
I don't understand this check either.

Reply:
This part would be confusing.
At the time of distribution, if the recipient balance after distribution exceeds MaxDistributedCapital, then we skip distribution at that particular time. Since it is a continuous distribution, there should be no limit on the funds to be distributed until the request has been cancelled.
Maybe marko can explain it more better 
cc: @tac0turtle

Reply:
maybe we dont need this check. The proposal would be for a % of the community pool, the max may be confusing since it doesnt add much value since the proposal wasnt for a max amount but a percentage. the max amount would be an off chain coordination thing.

Reply:
its a weird trade off reading through the code, the only reason we would want a max is if the max is hit we allocate funds back to others, meaning there isnt funds waiting for them to be claimed once they clear the account

Reply:
Personally, I understand it better, and indeed my suggestion for keeping the funds aside then does not make sense.
This should be reverted: https://github.com/cosmos/cosmos-sdk/commit/e0f0d2d14d42e559dd20bd683cc7a9bc2da28886.
However, all the implications should be made very clear in the documentation.

Reply:
Then, I'd revert what julien suggested and also remove `maxDistributedCapital`